### ISSUE TITLE
A problem when training on IEMOCAP dataset

### ISSUE THREAD

**Issue by zbxytx** [shenwzh3/DialogXL#3]
![1cfb19d4fdceb9122b5bc3e31bea99e](https://user-images.githubusercontent.com/22320221/108357468-6f824a00-7228-11eb-9a5b-4309d1e08f82.png)

While training DialogXL on IEMOCAP dataset, the accuracy and f-score didn't increase. I found that some people faced the same problem, so can you check the files or codes used in the training stage on IEMOCAP? 

(By the way, while trainig on MELD dataset, the model works correctly）

**Comment by Digimonseeker** [comment]
Sorry for this mistake，we have updated the IEMOCAP files.

**Comment by zbxytx** [comment]
With the new files you provided, the problem isn't be fixed. Can you reproduce the results with the existing files?

**Comment by shenwzh3** [comment]
Hi, you may check out our new version of code, it is now compatible with the Transformer v4.3.3. We have tested the code locally, the hyper-parameters worked well in this version.

**Comment by so-hyeun** [comment]
The same phenomenon occurs when running the train IEMOCAP using the newly uploaded version and Transformer v4.3.3.
Is there any updated code besides the readme?

**Comment by SteveKGYang** [comment]
Seems something wrong with the learning rate. I change it to 1e-6 and the training process worked. But the result in the paper is still not reproduced(F1 around 63). Could you check the hyper-parameters?
